apis/v1alpha3/backendtlspolicy_types.go

```diff
@@ -71,6 +71,8 @@ type BackendTLSPolicySpec struct {
 	//
 	// +kubebuilder:validation:MinItems=1
 	// +kubebuilder:validation:MaxItems=16
+	// +kubebuilder:validation:XValidation:message="sectionName must be specified when targetRefs includes 2 or more references to the same target",rule="self.all(p1, self.all(p2, p1.group == p2.group && p1.kind == p2.kind && p1.name == p2.name ? ((!has(p1.sectionName) || p1.sectionName == '') == (!has(p2.sectionName) || p2.sectionName == '')) : true))"
```

Comment:
Does this CEL apply itself appropriately when there are more than 2 `targetRefs`?  Maybe add another test case to verify that.

Reply:
sure, I added some additional `targetRefs` in the test cases